classes/security/authorization/internal/ApiTokenDecodingMiddleware.inc.php

```diff
@@ -36,27 +38,58 @@ public function __construct(APIHandler $handler) {
 	 */
 	protected function _decode($slimRequest) {
 		$secret = Config::getVar('security', 'api_key_secret', '');
+
+		if ($secret === '') {
```

Comment:
Let's check `if (!$secret)` here and below to catch cases where someone might have a stray space or tab character.